src/components/Core.vue

```diff
@@ -1,15 +1,10 @@
 <template>
   <main class="main">
     <div class="container mt-5">
-      <section class="top-column column">
-        <h1 class="title is-size-4 has-text-grey has-text-weight-medium mb-4">Bienvenue {{userProfile ? userProfile.name : 'inconnu'}}</h1>
-      </section>
-      <!-- conteneur des colonnes de gauche et droite où se situent les cards -->
```

Comment:
Nice, there still seem to have a lot more code you can remove in this commit and same very file. Am I wrong ? (computed properties and some part of the `mounted` callback.

Reply:
You're right, but again, I think this is addressed in another PR and I was afraid to mix things up. I'll try and clean the useless code in this one as you requested.

Reply:
Cleaned